src/remote/activitypub/models/person.ts

```diff
@@ -330,6 +331,7 @@ export async function updatePerson(uri: string, resolver?: Resolver | null, hint
 		lastFetchedAt: new Date(),
 		inbox: person.inbox,
 		sharedInbox: person.sharedInbox || (person.endpoints ? person.endpoints.sharedInbox : undefined),
+		followersUri: person.followers,
```

Comment:
I think it's better to do the same as createPerson above.

Reply:
good catch, thank you!